code-challenges.test.js

```diff
@@ -11,38 +13,135 @@
+    else if(fibLength2 < 2) {
+    return [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+}
+}    
+
+//Psuedocode:
+// 1. Declare a function called fibonacci
+// 2. Create a parameter called sequene
+// 3. Create conditional statments if/else to do the greater than logic
+// 4. Create return statments for each variable with the expected output
 
+//---------------Notes question 1---------------------
+// For the life of me I can't get the syntax right, unsure on what's going wrong there.
+// error message: Details:
+
+//     SyntaxError: /Users/learnacademy/Desktop/week-3-assessment-VenessaSP/code-challenges.test.js: Unterminated string constant. (17:7)
+
+//     15 |
+//     16 | describe ("fibonacci", () => {
+//   > 17 |     it("should return an array of the Fibonacci sequence”, () => {
+//        |        ^
+//     18 |         const fibLength1 = 6
+//     19 |         const fibLength2 = 10
+//     20 |     expect(fibonacci(fibLength1)
+
+   
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
-// a) Create a test with expect statements for each of the variables provided.
+describe("oddNumbers", () => {
+    it("takes in an array and returns a new array of only odd numbers sorted from least to greatest", () => {
+        const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+        const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+        expect(oddNumbers(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(oddNumbers(fullArr2)).toEqual([-823, 7, 23])
+})
+})
+
+
+// // a) Create a test with expect statements for each of the variables provided.
 
 const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
+// // Expected output: [-9, 7, 9, 199]
 
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+// // Expected output: [-823, 7, 23]
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+const oddNumbers = (nums) => {
+    for(let i=0; i<nums.length; i++){
+        if(nums[i] % 2 === 1) {
+            return []
+        }
+
+}
+}
+//Psuedocode:
+// 1. Declare a function called oddNumbers
+// 2. Create a parameter called nums
+// 3. Create a for loop with a modulo to get a remainder which would result in the odd numbers
+// 4. Create return statment
+
+// ----------------Notes question 2-------------------
+// unsure about the numbers in the strings, is the question looking for strictly data type numbers or string is included? unsure of what method to use as far as including strings or being able to sort from smallest to largest, I was thinking in the return line just write out all the odd numbers in that order or a .split & .join method but unsure on what to add for the reassignment. Can one parameter be used for 2 variables? 
+
```

Comment:
You're off to a good start with the logic!  You need to find the method that checks for the typeof data to separate numbers from numbers as strings.  Remember that you don't want to hardcode the return.  There are a few ways to get the end result, but I would recommend looking a HOF that can filter out the odds.